source/agora/utils/SCPPrettyPrinter.d

```diff
@@ -0,0 +1,606 @@
+/*******************************************************************************
+
+    Contains pretty-printing routines specialized for SCP types.
+
+    As the integration test-suite imports into PrettyPrinter,
+    we want to avoid adding a dependency to SCP.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.utils.SCPPrettyPrinter;
+
+import agora.common.Types;
+import agora.common.Amount;
+import agora.common.Config;
+import agora.common.Serializer;
+import agora.common.Types;
+import agora.common.crypto.Key;
+import agora.consensus.data.Block;
+import agora.consensus.data.ConsensusData;
+import agora.consensus.data.Enrollment;
+import agora.consensus.data.Transaction;
+import agora.utils.PrettyPrinter;
+
+import scpd.Cpp;
+import scpd.types.Stellar_SCP;
+import scpd.types.Stellar_types : StellarHash = Hash;
+alias StellarValue = Value;
```

Comment:
Why does one use renamed import and the other uses (public) alias ? Personally I prefer the former.